README.md

```diff
@@ -192,7 +192,9 @@ The following environment variables can be set:
 | `OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_MODE`                          | `Manual`                                                                 |
 | `OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY`              | `./.open-space-toolkit/physics/coordinate/frame/providers/iers`          |
 | `OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY_LOCK_TIMEOUT` | `60`                                                                     |
-| `OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_REMOTE_URL`                    | `https://maia.usno.navy.mil/ser7/`                                       |
+| `OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_BULLETIN_A_MANAGER_REMOTE_URL`                    | `https://media.githubusercontent.com/media/open-space-collective/open-space-toolkit-data/main/data/coordinate/frame/providers/iers/bulletin-A/`                                       |
```

Comment:
The two frame data files, BulletinA and Finals2000A, now have different remote URLs. Our API assumes that they have the same URL. To get around this, I made the following changes:
- Define new env vars + get/set methods for each separately (e.g. `getBulletinARemoteUrl`)
- The Environment variable `OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_REMOTE_URL` now overrides both URLs at the same time. (Removing it would technically be a breaking change)
- The original getter/setter methods (e.g. `getRemoteUrl()`) now redirect to the ones for BulletinA, since it is our preferred data source (e.g. `getBulletinARemoteUrl()`)

These changes are strictly for how we handle the URL